operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainFailureSeverity.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.weblogic.domain.model;
+
+public enum DomainFailureSeverity {
+  FATAL("Fatal"), SEVERE("Severe"), WARNING("Warning");
```

Comment:
This will probably fail to deserialize from JSON properly without the `@SerializedName` annotation on each enum value similarly to what we have in DomainConditionType.

Reply:
That's possible; it could explain the failures. I will fix that, thanks.